Languages/English/Keyed/AllKeyed_AutoMachineTool.xml

```diff
@@ -18,7 +18,7 @@
   <PRF.AutoMachineTool.OutputDirectionW>West</PRF.AutoMachineTool.OutputDirectionW>
   <PRF.AutoMachineTool.OutputDirectionNW>Northwest</PRF.AutoMachineTool.OutputDirectionNW>
 
-  <PRF.AutoMachineTool.PlaceNotAllowed>AutoMachineTool must face work table and be placed in working position.</PRF.AutoMachineTool.PlaceNotAllowed>
+  <PRF.AutoMachineTool.PlaceNotAllowed>AutoMachineTool must face the work table and be placed in a working position.</PRF.AutoMachineTool.PlaceNotAllowed>
```

Comment:
are there any buildings with Multiple  working positions? 
Would `the` be better here?

Reply:
I think I was not sure at the moment. I will change it to "the" then.